interface/presenter/pokemon_presenter.go

```diff
@@ -17,5 +17,10 @@ func NewPokemonPresenter() PokemonPresenter {
 }
 
 func (pp *pokemonPresenter) ResponsePokemons(pokemons []*model.Pokemon) []*model.Pokemon {
+
```

Comment:
If you only have one import, do it in a single line
```go
import "github.com/luispaulin/api-challenge/domain/model"

```

Reply:
Also, remove this empty line

Reply:
Done